UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs

```diff
@@ -11,6 +11,9 @@ namespace Gateway
 	/// </summary>
 	public class TransportUtility : NetworkBehaviour //Would be a regular static class, but Weaver complains if it doesn't inherit NetworkBehaviour
 	{
+		private static readonly Vector3 maintRoomsLocation = new Vector3(3005,-2916,0);
```

Comment:
isn't there any better way of getting the position of the maint rooms such as a component on a prefab , like abandoned teleporter in the maintenance rooms

Reply:
The backrooms are only loaded some of the time,. Could at a marker object but that would also require an additional script on that object. I think its cleaner just to have a constant?

Reply:
If it Wasn't loaded then Wouldn't you just teleport into the middle of space where it should be?

Reply:
Theres a check. SubSceneManager.Instance.IsMaintRooms